test/e2e/scripts/run-instance/11-setup-kind-cluster.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+printf '=%.0s' {0..79} ; echo
+set -x
+
+echo "Setup kind cluster: e2e-k8s-test"
+
+kind create cluster --name k8s-e2e-tests --wait 10m
```

Comment:
`e2e-k8s-test` vs. `k8s-e2e-tests`
`e2e` and `k8s` swapped between the log message and the `kind` command.